### PR TITLE
Changed master nodes to controlplane

### PR DESCRIPTION
**Pull request by koneill94**
Masters are now deployed as controlplane nodes. Tested a deployment.

### CODE REVIEW

File: common.yaml [hunk 1]
```diff
@@ -20,19 +20,19 @@
       os_port_master: "{{ infraID }}-master-port"
       os_port_worker: "{{ infraID }}-worker-port"
       # Security groups names
-      os_sg_master: "{{ infraID }}-master"
+      os_sg_master: "{{ infraID }}-controlplane"
```
Comment (stevemul) [inline common.yaml:23]:
```suggestion
      os_sg_controlplane: "{{ infraID }}-controlplane"
```

Comment (stevemul) [reply common.yaml:23]: I'm sure this change would cascade through elsewhere, but if we're changing it in one place might as well remove the term throughout the code base if we can?

File: master.sh [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    MASTER_HOSTNAME="$INFRA_ID-controlplane-$index\n"
     python -c "import base64, json, sys;
 ignition = json.load(sys.stdin);
 storage = ignition.get('storage', {});
 files = storage.get('files', []);
 files.append({'path': '/etc/hostname', 'mode': 420, 'contents': {'source': 'data:text/plain;charset=utf-8;base64,' + base64.standard_b64encode(b'$MASTER_HOSTNAME').decode().strip(), 'verification': {}}, 'filesystem': 'root'});
 storage['files'] = files;
 ignition['storage'] = storage
-json.dump(ignition, sys.stdout)" <master.ign >"$INFRA_ID-master-$index-ignition.json"
+json.dump(ignition, sys.stdout)" <master.ign >"$INFRA_ID-controlplane-$index-ignition.json"
```
Comment (stevemul) [inline master.sh:10]: I assume master.ign is a filename that the installer generates?

File: server-groups.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@
   - name: Create master server group
```
Comment (stevemul) [inline server-groups.yaml:19]:
```suggestion
  - name: Create control plane server group
```

**Review by stevemul — CHANGES_REQUESTED** [review]
Some more areas that need changed - might as well clean the whole lot up when we're doing this for the sake of consistency if possible.

**Review by gellner — APPROVED** [review]
LGTM

**Comment by koneill94** [comment]
@stevemul Can you take another look? Changes made in response to your comments.

**Review by stevemul — APPROVED** [review]
Provided it's been tested and works still then LGTM.

**Pull request merged**